tests/testthat/test-RtGam.R

```diff
@@ -1,10 +1,26 @@
 test_that("RtGam parses inputs successfully", {
   timesteps <- 20
-  cases <- 1:timesteps
+  withr::with_seed(12345, {
```

Comment:
as it stands i would have this in suggests

Reply:
https://github.com/CDCgov/cfa-gam-rt/pull/22/commits/a841218d8dd92a6beb12c93e753e311e6789ad46